### ISSUE TITLE
Not compatible with Django 4.2.1

### ISSUE THREAD

**Issue by malekzahedi** [miki725/django-url-filter#119]
I installed django-url-filter  0.3.15 and Django 4.2.1 and got the following error:
`ImportError: cannot import name 'ungettext_lazy' from 'django.utils.translation'`

Line 9 of validators.py should be updated to: 
`from django.utils.translation import gettext_lazy`

**Comment by enjoy2000** [comment]
yes check this issue please https://github.com/miki725/django-url-filter/issues/113

**Comment by MoncefB** [comment]
FYI I've [forked this repo](https://github.com/MoncefB/django-rest-url-filter) as I really needed to upgrade Django and started losing hope..
If that's useful to anyone else, you can give `pip install django-rest-url-filter` a shot! :)

**Comment by feroda** [comment]
> FYI I've [forked this repo](https://github.com/MoncefB/django-rest-url-filter) as I really needed to upgrade Django and started losing hope.. If that's useful to anyone else, you can give `pip install django-rest-url-filter` a shot! :)

thanks a lot, 
I tell you there is a typo in your README.md `pip install django-rest-url-filter` and not `pip install url-filter`
